src/handlers/major_change.rs

```diff
@@ -210,8 +210,9 @@ pub(super) async fn handle_input(
             }) {
                 // Re-schedule acceptance job to automaticaly close the MCP
                 schedule_acceptance_job(ctx, config, &event.issue).await?;
+                let waiting_period = config.waiting_period.unwrap_or(10);
```

Comment:
The default `10` is now configured at two separate places. Either create a function for this, or just turn `waiting_period` from `Option<u16>` to `u16` and configure the default using serde?

Reply:
I don't yet want to use serde default, as to would instantaneously enable it which it shouldn't, I will create a method.

Reply:
Create a `displayable_waiting_period` method.

Reply:
I'm not I understand. This "10 days" is purely for display, we don't do any calculation with it.

This change is about keeping the current display, while using the new waiting period config field if specified, and we currently say 10 days everywhere.

Reply:
> I don't yet want to use serde default, as to would instantaneously enable it which it shouldn't, I will create a method.

Actually, this confuses me. The code only does auto accept if `waiting_period` is `Some`, but here the message about accepting is always sent. Does that mean that `Some` means "accept happens automatically" and `None` means "accept happens manually"? What if I want to accept manually with a different waiting period? :D I think we should separate the waiting period from enabling auto accept.

Reply:
> Does that mean that `Some` means "accept happens automatically" and `None` means "accept happens manually"?

Yes, that's how the field is documented.

> What if I want to accept manually with a different waiting period?

Not a scenario I envisioned. The only reason the waiting period is even configurable is because I didn't wanted to wait 10 days for testing it (but then I still hacked it to 5 minutes in the end ;-)).

> I think we should separate the waiting period from enabling auto accept.

Done, I added a new `auto_closing` field, it defaults to false.

Reply:
It's likely a use-case that no one's gonna ever hit, so sorry for bothering you with it :laughing: I still think that this is a clearer design, so thank you.